packages/aws-cdk-lib/aws-ecs-patterns/README.md

```diff
@@ -582,6 +582,19 @@ const queueProcessingFargateService = new ecsPatterns.QueueProcessingFargateServ
 });
 ```
 
+### Set Cooldown parameter for QueueProcessingFargateService
```

Comment:
Missing one sentence explaining what the cooldown parameter is

Reply:
added sentence below, let me know if it needs changing